### ISSUE TITLE
HTML/Javascript naming conflict for form validation

### ISSUE THREAD

**Issue by mslomski** [evanlws12/423-Term-Project#33]
So when I was creating form validation for the insert item feature I noticed that the item's description field is named "description". In javascript this is a keyword that may conflict with the validation process when trying to access that element. We should probably change the name for the element to something like "itemDescription".

Also for the validation of promotions the promotion name is given the element name "name", which is also a keyword in javascript.

In order to make the life of validation far more easier, we need to change these names to something more functional.

**Comment by mslomski** [comment]
As the results of the changes above, the PHP would have to be updated to accommodate the transfer of data with the new names.

**Comment by mslomski** [comment]
Another thing I noticed is that the insert promotion form does not have a name or id attached like the insert item form. I feel that it is a good idea to have this for validation purposes. So I feel that we should ultimately make sure we add these to the process for every form we validate.

Also I found another naming problem with the description of insert promotion. Make sure you look at all the form names and that it meets the standards.
